airflow/api_connexion/endpoints/connection_endpoint.py

```diff
@@ -125,7 +125,6 @@ def patch_connection(
     for key in data:
         setattr(connection, key, data[key])
     session.add(connection)
-    session.commit()
     return connection_schema.dump(connection)
```

Comment:
This one (and all code similar to this where there’s a `dump` after `commit`) cannot be safely removed. Saving the instance to database can trigger value updates (e.g. autoincrement columns), so `commit` must happen before `dump` to ensure those values are correctly populated.